playbooks/build-ansible-collection/run.yaml

```diff
@@ -31,6 +31,11 @@
         patterns: "*.tar.gz"
       register: result
 
+    - name: Install the Collections locally, to be sure the dependencies get pulled
+      command:  "{{ ansible_galaxy_executable }} collection install {{ item.path }}"
+      loop: "{{ result.files }}"
+      ignore_errors: yes
+
```

Comment:
Trying to think of a better way to do this, we don't want to ignore_errors, otherwise we'll never know something failed. This also now means we depend directly on galaxy over source depends-on.  Given this is only an issue with 3pci jobs, I'd like us to have a flag, and projects opt into it.

The proper fix, IMO, is ability to disable doc checks for cross-projects, as having them _installed_ seems wrong for galaxy-importer to work.

Reply:
I think it's actually safe to enable it everywhere. My artifacts have been built using the Depends-On:, and i install using `ansible-galaxy collection install --force foo.tar.gz`, I install the collections that I've built. If one of my collection is already installed with a version from Galaxy it will be overwritten.
We can also play with `ANSIBLE_COLLECTIONS_PATHS` to install everything in a temporary directory, but I don't think it's necessary.

e.g:
```
ansible-galaxy collection install --force /home/goneri/git_repos/ansible-collections/community.general/community-general-0.1.1.tar.gz
Process install dependency map
[WARNING]: Failed to validate the collection requirement 'ansible.netcommon:>=0.0.2' for community.general when the existing install does not have a version set, the collection may not
work.
Starting collection install process
Installing 'community.general:0.1.1' to '/home/goneri/.ansible/collections/ansible_collections/community/general'
Skipping 'ansible.netcommon' as it is already installed
Skipping 'ansible.posix' as it is already installed
Skipping 'community.kubernetes' as it is already installed
Skipping 'google.cloud' as it is already installed
```

Reply:
One case may be problematic. I push a change to `galaxy.yaml` to add a dependency on another collection that is not on Galaxy yet. Can I use the `Dependes-On:` in this case to ask Zuul to pull that version and prepare the collection .tar.gz?